plugins/uberdev/skills/testers-pipeline/report.py

```
@@ -0,0 +1,128 @@
+import os
+import sys
+import yaml
+
+
+FILEABLE = {"blocker", "critical", "major"}
+
+
+def load_waves(waves_dir: str) -> list[dict]:
+    out = []
+    for p in sorted(glob.glob(os.path.join(waves_dir, "wave-*.yaml"))):
+        out.append(yaml.safe_load(open(p)))
```

Ruff (0.15.13)

[warning] Use a list comprehension to create a transformed list

(PERF401)